airframe-sql/src/main/scala/wvlet/airframe/sql/parser/SQLGenerator.scala

```diff
@@ -374,7 +374,7 @@ object SQLGenerator extends LogSupport {
       case l: Literal =>
         l.sqlExpr
       case g: GroupingKey =>
-        g.sqlExpr
+        g.index.getOrElse(printExpression(g.child)).toString
```

Comment:
`g.index.map(_.toString).getOrElse(printExpression(g.child))` is better for the readability.

Reply:
Ahhh 💡. I think I understand how the SQL generation is structured. I reverted the changes to this line and adjusted the implementation of `sqlExpr` for the new `Extract` case class.